common/djangoapps/student/views/dashboard.py

```diff
@@ -629,6 +632,21 @@ def student_dashboard(request):
 
     enterprise_message = get_dashboard_consent_notification(request, user, course_enrollments)
 
+    recovery_email_message = None
+
+    # Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    is_enterprise_learner_data = is_enterprise_learner(user=request.user)
+    does_secondary_email_exist = AccountRecovery.objects.filter(user=user).values_list('secondary_email')
```

Comment:
`secondary_email` is a non nullable and required column of `AccountRecovery` so you can simply do this.
```
does_secondary_email_exist = AccountRecovery.objects.filter(user=user).exists()
```